### DISCUSSION TITLE
ghas-will-win-the-appsec-wars/

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#16] [Giscussions]
# ghas-will-win-the-appsec-wars/

GitHub Advanced Security is positioned to win the “AppSec Wars”. In this post I go over why I think this is the case.

https://colinsalmcorner.com/ghas-will-win-the-appsec-wars/

**Comment by MattCase1** [comment]
Awesome post! Super helpful in providing a holistic view of the current state of AppSec (traditional Security approach) and what organizations' future state could look like.
